### PR TITLE
EllipsoidGeodesic Updated with Check

### PR DESCRIPTION
**Pull request by Rudraksha20**
Worked on issue #4794 updated EllipsoidGeodesic.js, removed DeveloperErrors to include Check.

### CODE REVIEW

**Comment by pjcozzi** [comment]
Very nice, thanks @Rudraksha20!

@omh1280 can you please review this?  Note that `DeveloperError` no longer needs to be included in the changed file.

**Comment by pjcozzi** [comment]
@Rudraksha20 can you also please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla)?

File: Source/Core/EllipsoidGeodesic.js [hunk 1]
```diff
@@ -9,6 +10,7 @@ define([
         './Ellipsoid',
```
Comment (omh1280) [inline Source/Core/EllipsoidGeodesic.js]: As Patrick said, please remove `'./DeveloperError',` and the corresponding variable `DeveloperError`. Thanks!

File: Source/Core/EllipsoidGeodesic.js [hunk 2]
```diff
@@ -179,9 +181,7 @@ define([
         var lastCartesian = Cartesian3.normalize(ellipsoid.cartographicToCartesian(end, scratchCart2), scratchCart2);
 
         //>>includeStart('debug', pragmas.debug);
-        if (Math.abs(Math.abs(Cartesian3.angleBetween(firstCartesian, lastCartesian)) - Math.PI) < 0.0125) {
-            throw new DeveloperError('geodesic position is not unique');
-        }
+        Check.typeOf.number.greaterThanOrEquals('value', Math.abs(Math.abs(Cartesian3.angleBetween(firstCartesian, lastCartesian)) - Math.PI), 0.0125)
```
Comment (omh1280) [inline Source/Core/EllipsoidGeodesic.js]: Oh! Missing semicolon! Make sure to run `npm run eslint` before pushing.

File: Source/Core/EllipsoidGeodesic.js (hunk 1 above)
Comment (Rudraksha20) [inline Source/Core/EllipsoidGeodesic.js]: Thanks @pjcozzi @omh1280, I have made the changes.

**Comment by pjcozzi** [comment]
Congrats on your first pull request!

Thanks for the review, @omh1280.

File: Source/Core/EllipsoidGeodesic.js (hunk 2 above)
Comment (Rudraksha20) [inline Source/Core/EllipsoidGeodesic.js]: oops! thanks @omh1280 will make sure to run it next time.

**Pull request merged**